selfdrive/ui/ui.h

```diff
@@ -24,7 +24,19 @@ const int UI_FREQ = 20; // Hz
 const int BACKLIGHT_OFFROAD = 50;
 typedef cereal::CarControl::HUDControl::AudibleAlert AudibleAlert;
 
-const mat3 DEFAULT_CALIBRATION = {{ 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0 }};
+const int TRAJECTORY_SIZE = 33;
```

Comment:
these constants still belongs in selfdrive/modeld. can we export the python ones into a header at build time instead?

Reply:
removed TRAJECTORY_SIZE, use size from position list